frontend/src/scenes/billing/UnsubscribeSurveyModal.scss

```diff
@@ -0,0 +1,10 @@
+.UnsubscribeSurveyCostControl {
```

Comment:
We should try and do this with Tailwind, I know the team is moving away from `.scss`.

You can use `!` for important in Tailwind: https://tailwindcss.com/docs/configuration#important

Reply:
I tried using [arbitrary variants](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-variants) to implement the same thing and it doesn't seem to have any effect. Something like 
```[.LemonCollapsePanel__header]:!p-0:bg-bg-3000_[.LemonButton__chrome]:!p-0```

Reply:
@Twixes are you the right person for tailwind help?

Reply:
You could update the `LemonCollapse` component to allow `headerClassName` .

Reply:
Hey, sorry, I totally missed this mention! Just took a look at the PR, but it looks like `.UnsubscribeSurveyCostControl` is dead code?